docs/website/docs/developers/general/release-management.md

```diff
@@ -32,9 +32,9 @@ When selecting a candidate we use the following criteria:
 2. Contains no P0 regressions vs the previous stable release
 3. LLVM submodule commit ideally exists upstream (no cherry picks or patches)
 
-When you've identified a potential candidate, email the iree-discuss list with
-the proposal and solicit feedback. People may point out known regressions or
-request that some feature make the cut.
+When you've identified a potential candidate, email the
+iree-technical-discussion list with the proposal and solicit feedback. People
+may point out known regressions or request that some feature make the cut.
```

Comment:
FYI I'll be updating this process as part of https://github.com/iree-org/iree/issues/18380 / future releases. Might still be useful to email the list, but just commenting on the GitHub issue will serve the same purpose.

Reply:
Thanks for the pointer. I think it should be fine to update here and to cleanup with #18380?